tasks/build.yml

```diff
@@ -22,8 +22,10 @@
   become: yes
   become_user: "{{ sansible_kafka_user }}"
   get_url:
-    dest: "/home/{{ sansible_kafka_user }}/tmp/kafka_{{ sansible_kafka_version_scala }}-{{ sansible_kafka_version_kafka }}.tgz"
-    url: "{{ sansible_kafka_apache_mirror }}/kafka/{{ sansible_kafka_version_kafka }}/kafka_{{ sansible_kafka_version_scala }}-{{ sansible_kafka_version_kafka }}.tgz"
+    dest: "/home/{{ sansible_kafka_user }}/tmp/\
+      kafka_{{ sansible_kafka_version_scala }}-{{ sansible_kafka_version_kafka }}.tgz"
+    url: "{{ sansible_kafka_apache_mirror }}/kafka/{{ sansible_kafka_version_kafka }}/\
+      kafka_{{ sansible_kafka_version_scala }}-{{ sansible_kafka_version_kafka }}.tgz"
```

Comment:
Don't break the lines here... that's really ugly :-)

Reply:
that's to fix lintting error 

```ASCII
--> Executing Ansible Lint on /Volumes/User/workspace/voytek/sansible/kafka/molecule/default/playbook.yml...
    [204] Lines should be no longer than 160 chars
    /Volumes/User/workspace/voytek/sansible/kafka/tasks/build.yml:26
        url: "{{ sansible_kafka_apache_mirror }}/kafka/{{ sansible_kafka_version_kafka }}/kafka_{{ sansible_kafka_version_scala }}-{{ sansible_kafka_version_kafka }}.tgz"
```